py/server/deephaven/pandas.py

```diff
@@ -62,8 +62,27 @@ def _column_to_series(table: Table, col_def: Column) -> pandas.Series:
     pa.timestamp('ns', tz='UTC'): pd.ArrowDtype(pa.timestamp('ns', tz='UTC')),
 }
 
+_DTYPE_MAPPING_NUMPY_NULLABLE = {
+    pa.int8(): pd.Int8Dtype(),
+    pa.int16(): pd.Int16Dtype(),
+    pa.int32(): pd.Int32Dtype(),
+    pa.int64(): pd.Int64Dtype(),
+    pa.bool_(): pd.BooleanDtype(),
+    pa.float32(): pd.Float32Dtype(),
+    pa.float64(): pd.Float64Dtype(),
+    pa.string(): pd.StringDtype(),
+    # pa.Table.to_pandas() doesn't support mapping to pd.DatetimeTZDtype,
+    # pa.timestamp('ns'): pd.DatetimeTZDtype(unit='ns', tz='UTC'),
+    # pa.timestamp('ns', tz='UTC'): pd.DatetimeTZDtype(unit='ns', tz='UTC'),
```

Comment:
I'm surprised that times are not nullable.  A little digging indicates that there is a `pa.NaT`.
https://pandas.pydata.org/docs/reference/api/pandas.to_datetime.html
Does this mean that timestamps *are* nullable?

Reply:
Yes, that's exactly what pyarrow converts it to.